.buildkite/pipeline.yml

```diff
@@ -68,29 +68,29 @@ steps:
           - target/libwordpressFFI.xcframework.zip
           - native/swift/Sources/wordpress-api-wrapper/wp_api.swift
         env:
-          IMAGE_ID: xcode-16.0-v7
+          IMAGE_ID: $IMAGE_ID
```

Comment:
Nice DRYing!

💄 Nowadays we usually also move the `env:` section for `IMAGE_ID: $IMAGE_ID` at the top of the `pipeline.yml` files so that it's common to all steps (sure that means that env var is also passed to steps that use agents other than `mac` and for which defining this `IMAGE_ID` is useless… but it's also harmless and allows some more DRYing, so…)

Reply:
> sure that means that env var is also passed to steps that use agents other than mac and for which defining this IMAGE_ID is useless… but it's also harmless and allows some more DRYing, so…

Yeah, that's what I considered in principle (passing IMAGE_ID to other agents that don't use it, in principle with no harm), but indeed it looks a lot cleaner. Updated on e8936045c164074c6015160827bb3caed5830cd4